application/migrations/20200107221947_create_watches.up.fizz

```diff
@@ -0,0 +1,10 @@
+create_table("watches") {
+	t.Column("id", "integer", {primary: true})
+	t.Column("uuid", "uuid", {})
+	t.Column("owner_id", "integer", {})
+	t.Column("location_id", "integer", {null: true})
+	t.ForeignKey("owner_id", {"users": ["id"]}, {"on_delete": "cascade"})
+	t.Foreign_key("location_id", {"locations": ["id"]}, {"on_delete": "set null"})
```

Comment:
Don't you want this to be "cascade" too?
If I'm understanding this, the watch starts by being associated with a Location. So, we will end up notifying that users when there is something happening in relation to that Location (e.g. a Post is created near it).

If for some strange reason that Location is deleted, then we would want this watch to be deleted also.
Otherwise, this watch might end up being valid for all locations or something, right?

Reply:
Yeah, that makes sense.